lib/Container/config/menu.json

```diff
@@ -98,6 +98,13 @@
                 "source_path": "%path%"
             }
         },
+        "overload_method": {
+            "action": "overload_method",
+            "parameters": {
+                "path": "%current_path%",
+                "source": "%source%"
```

Comment:
This shoud really be an offset - we should be able to determine the outer class from an offset.